doc/man3/SSL_CTX_set_options.pod

```diff
@@ -137,7 +148,9 @@ handshake). This option is not needed for clients.
 
 =item SSL_OP_NO_COMPRESSION
 
-Do not use compression even if it is supported.
+Do not use compression even if it is supported. This option is set by default.
+To switch it off use SSL_clear_options(). A future version of OpenSSL may not
+set this by default.
```

Comment:
It's not clear what's being implied.  A future version could allow compression by default or a future version might not allow compression to be enabled?

Reply:
`SSL_OP_MIDDLEBOX_COMPAT` has the same weaselly language. It allows for either. :)

Reply:
The rest has been updated. Did you want something for this?

Reply:
I would say, fix this one. In theory the MIDDLEBOX_COMPAT could be less and less necessary and we might switch it off by default. Where SSL compression is more likely to be completely removed in future (not certificate compression though which might be added). So please just remove the `A future version...` sentence.

Reply:
Yup, I'm working on #9155. I can remove this sentence.